processor/attributesprocessor/attributes_metric_test.go

```diff
@@ -56,6 +59,11 @@ func generateMetricData(resourceName string, attrs map[string]interface{}) pmetr
 	sl := res.ScopeMetrics().AppendEmpty()
 	m := sl.Metrics().AppendEmpty()
 
+	g := m.SetEmptyGauge()
+	dp := g.DataPoints().AppendEmpty()
+	dp.SetTimestamp(pcommon.NewTimestampFromTime(time.UnixMicro(0)))
```

Comment:
I would suggest not using `time = 0` since I am not sure if it would be equal to zero value `time.Time` which could match an unset `time.Time` (golang's zero value).

I don't think this will happen but it is something to considering when dealing with using fixed time.

Reply:
changed to an arbitrary value